stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDOMProvider.java

```diff
@@ -48,6 +50,24 @@
   private final InspectModeHandler mInspectModeHandler;
   private Listener mListener;
 
+  // We don't yet have an an implementation for reliably detecting fine-grained changes in the
+  // View tree. So, for now at least, we have a timer that runs every so often and just reports
+  // that we changed. Our listener will then read the entire DOM from us and transmit the changes to
+  // Chrome. Detecting, reporting, and traversing fine-grained changes is a future work item.
+  private boolean mIsReportChangesTimerPosted = false;
+  private final Runnable mReportChangesTimer = new Runnable() {
+    @Override
+    public void run() {
+      mIsReportChangesTimerPosted = false;
+
+      if (mListener != null) {
+        mListener.onChanged();
```

Comment:
No this doesn't compute any diffs, it's telling DOM when it should do that (or is that what you mean? this is what kicks off the process, in other words). I don't really like the name onPossiblyChanged(), but another verb other than "changed" may be useful here. The intent of the interface is that the DOMProvider should be "smart" about raising the onChanged() event, and it's kind of an implementation detail of AndroidDOMProvider that it isn't smart enough to suppress it. 

My original thinking on this protocol lined it up with the "invalidate" pattern seen elsewhere, e.g. `InvalidateRect` https://msdn.microsoft.com/en-us/library/windows/desktop/dd145002(v=vs.85).aspx . "Tell me when and where you need to be repainted. I'll debounce it and be smart about how I prioritize calling your paint code, balancing quick updates to the user vs. bogging down the system with it. It's okay if your new output is bit-identical to our old output, but it's also to everyone's advantage if you're smart and only call this method when it's actually necessary."

I will scratch my chin for a bit on this...